Networking/Networking/Remote/ProductReviewsRemote.swift

```diff
@@ -26,23 +26,24 @@ public final class ProductReviewsRemote: Remote, ProductReviewsRemoteProtocol {
     ///     - pageNumber: Number of page that should be retrieved.
     ///     - pageSize: Number of Orders to be retrieved per page.
     ///     - products: Limit result set to reviews assigned to specific product IDs.
+    ///     - filterByStatuses: Array of statuses to filter reviews by.
```

Comment:
This is an array because filtering by multiple statuses is supported. I'm copying FluxC's information here:

https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/14fcce73130015f14125e9e13837f84ff4171bb3/plugins/woocommerce/src/main/kotlin/org/wordpress/android/fluxc/network/rest/wpcom/wc/product/ProductRestClient.kt#L1584